### PR TITLE
Update dist-git merge-request

### PR DESCRIPTION
**Pull request by spotlesstofu**
Update the dist-git merge-request when the source-git merge-request is updated with new code.

1. Get `oldrev` from the event
2. If `oldrev` is defined , the merge-request has been updated with new code
3. Extract the `sync_release` function from the `run` function
4. Run `sync_release` when `oldrev` is defined

Merge after  packit/packit#1725

### CODE REVIEW

**Review by lachmanfrantisek — COMMENTED** [review]
Thanks!

Would it be possible to create a test case for this? (We can help and give pointers for sure..;)

I am leaving the final word to @csomh / @jpopelka as the most experienced ones in source-git workflow.

File: hardly/handlers/distgit.py [hunk 1]
```diff
@@ -130,6 +131,19 @@ def packit(self) -> PackitAPI:
             )
         return self._packit
 
+    def sync_release(self):
+        return self.packit.sync_release(
+            dist_git_branch=self.target_repo_branch,
+            version=self.packit.up.get_specfile_version(),
+            add_new_sources=False,
+            title=self.mr_title,
+            description=f"{fix_bz_refs(self.mr_description)}\n\n---\n{dg_mr_info}",
```
Comment (csomh) [inline hardly/handlers/distgit.py:140]: `dg_mr_info` is not defined in this scope.

Comment (spotlesstofu) [reply hardly/handlers/distgit.py:148]: Right. I moved all the dg_mr_info references in the sync_release function

**Review by jpopelka — COMMENTED** [review]
I haven't tried it but it should work I think. Thanks!

**Review by jpopelka — APPROVED** [review]
I've updated the commit to not use the `update_pr` argument because I had replaced packit/packit#1723 with packit/packit#1725 which doesn't need it.

I [tested this on stg and it looks OK](https://gitlab.com/packit-service/rpms/open-vm-tools/-/merge_requests/33).

Thank you!

**Pull request merged**